contracts/interchain-router/README.md

```diff
@@ -0,0 +1,7 @@
+# Interchain Router
+
+Interchain Router is a contract that facilitates a predetermined routing of funds.
+Each instance of the interchain router is associated with a single receiver.
+
+The router continuously attempts to perform IBC transfers to the receiver.
```

Comment:
You should mention that it tries to send all the balances for all demons in the contract account